src/cli/cli_tcp.cpp

```diff
@@ -377,49 +378,73 @@ template <> otError TcpExample::Process<Cmd("benchmark")>(Arg aArgs[])
 {
     otError error = OT_ERROR_NONE;
 
-    VerifyOrExit(!mSendBusy, error = OT_ERROR_BUSY);
-    VerifyOrExit(mBenchmarkBytesTotal == 0, error = OT_ERROR_BUSY);
-
-    if (aArgs[0].IsEmpty())
+    if (aArgs[0] == "result")
     {
-        mBenchmarkBytesTotal = OPENTHREAD_CONFIG_CLI_TCP_DEFAULT_BENCHMARK_SIZE;
```

Comment:
We can have a common `OutputFormat("TCP Benchmakr Status: ")` at the top and then shorten the `OutputLine()` below to show the status?